config/v1/0000_10_config-operator_01_infrastructure.crd.yaml-patch

```diff
@@ -0,0 +1,24 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/vcenters/items/properties/server/anyOf
+  value:
+    - format: ipv4
```

Comment:
@Miciah you've spent a lot of time in regexes and kubebuilder "validation".  Do these work?